### PR TITLE
Styled Text Part 2

### PR DESCRIPTION
**Pull request by mrios730**
Hi Dan! I was trying to use TinyMCE for the rich text editor, but I couldn't figure out how to use the library so I'm planning to work with CKEditor since they use that library as an example in the instructions. One of the steps says, "Put this line of code somewhere that runs after the body has been loaded. In user-page.html, calling this from the buildUI() function is fine." I'm confused as to where exactly to place this line of code ClassicEditor.create( document.getElementById('message-input') );
Right now, I have it commented out on the first line of the buildUI() function.

### CODE REVIEW

**Comment by dhgarrette** [comment]
Ah ha!  Those instructions are not clear, but I figured it out: [user-page.html calls the buildUI function](https://github.com/mrios730/Team-25/blob/master/src/main/webapp/user-page.html#L26) defined in [user-page-loader.js](https://github.com/mrios730/Team-25/blob/master/src/main/webapp/js/user-page-loader.js#L111), so you should add that `ClassicEditor.create(...)` call to the end of `buildUI` there:

```
function buildUI() {
  setPageTitle();
  showMessageFormIfLoggedIn();
  fetchMessages();
  fetchAboutMe();
  ClassicEditor.create( document.getElementById('message-input') );
}
```

**Comment by mrios730** [comment]
Thank you! I added the line to buildUI() and it works now :)

File: src/main/webapp/user-page.html [hunk 1]
```diff
@@ -22,8 +22,10 @@
     <link rel="stylesheet" href="/css/main.css">
     <link rel="stylesheet" href="/css/user-page.css">
     <script src="/js/user-page-loader.js"></script>
+    <script src="https://cdn.ckeditor.com/ckeditor5/11.2.0/classic/ckeditor.js"></script>
   </head>
   <body onload="buildUI();">
+    <!--ClassicEditor.create( document.getElementById('message-input') );-->
```
Comment (dhgarrette) [inline src/main/webapp/user-page.html]: You can remove this now.

File: src/main/webapp/user-page.html [hunk 2]
```diff
@@ -46,8 +48,6 @@ <h1 id="page-title">User Page</h1>
       <input type="submit" value="Submit">
     </form>
     <hr/>
-
```
Comment (dhgarrette) [inline src/main/webapp/user-page.html]: These three lines don't need to change, so you can leave them the way they were.

**Review by dhgarrette — APPROVED** [review]
Thanks, Maria!  A couple minor comments, but otherwise it looks good.

**Comment by dhgarrette** [comment]
Looks great!  Thanks, Maria!  You can go ahead and merge it.

**Pull request merged**